library/spdm_responder_lib/libspdm_rsp_chunk_get.c

```diff
@@ -103,12 +104,24 @@ libspdm_return_t libspdm_get_response_chunk_get(
             response_size, response);
     }
 
-    libspdm_zero_mem(response, *response_size);
-
     min_data_transfer_size = LIBSPDM_MIN(
         spdm_context->connection_info.capability.data_transfer_size,
         spdm_context->local_context.capability.sender_data_transfer_size);
 
+    /* Fail if exceed max chunks */
+    max_chunk_data_transfer_size =
```

Comment:
we should also consult max_spdm_msg_size.

Reply:
Same feedback as in https://github.com/DMTF/libspdm/pull/2952/files#r1943830275.